neurons/validators/src/protocol/vc_protocol/validator_requests.py

```diff
@@ -57,6 +57,11 @@ class ExecutorSpecRequest(BaseValidatorRequest):
     log_status: str | None
     job_batch_id: str
 
+class LogValidatorRequest(BaseValidatorRequest):
```

Comment:
![image](https://github.com/user-attachments/assets/0fad73da-6676-4913-b2a8-789fe230ef14)
I did like this. Is it okay?

Reply:
@pyon12

Reply:
@LegendWorrier you need to discuss the detailed data format with @talentwebdev , but we need to have the validator_hotkey, timestamp and also score/weights results here.

AND it would be good to get the `timestamp` from the bittensor api when setting weight like what we did for job_batch_id.

@talentwebdev ??